### DISCUSSION TITLE
Could I become a contributor/member?

### DISCUSSION THREAD

**Discussion by Mr-Programmer-777** [ScratchAddons/ScratchAddons#1788] [Help] [answered]
Yes, it is true I'm useless at everything code except Scratch and a little HTML, but I would be thankful to be able to help in different areas, like helping the community if there's a problem or answering questions. Another useful factor is that I'm online when a lot of members aren't. This could help with responding to people within 10 minutes instead of 5 hours.

Yours questioningly,
MrProgrammer777

**Comment by cobaltt7** [comment, marked as answer]
Anybody can help but AFAIK their not accepting new members

**Reply by Endermaster3690** [reply]
> Anybody can help but AFAIK their not accepting new members

what does AFAIK mean?

**Reply by cobaltt7** [reply]
as far as i know

**Reply by WorldLanguages** [reply]
Being a member mostly means nothing

**Comment by TheColaber** [comment]
Anyone can help.

**Comment by Hans5958** [comment]
You can. Anyone can. Read [How can I contribute to Scratch Addons?](https://scratchaddons.com/faq#how-can-i-contribute-to-scratch-addons)
